config/locales/chapters.yml

```diff
@@ -0,0 +1,85 @@
+en:
+  pages: 
+    chapters: 
+      headline: Start a Blueprint Chapter
+      subtitle:
+        Blueprint was founded at UC Berkeley in Spring 2013. 
+        Since then we started scaling our impact and further mission by establishing chapters in different colleges around the world. 
+      chapters:
+        - name: University of Waterloo  
+          description: UWaterloo is our first Blueprint chapter, founded in 2015 🍁
+          url: "https://uwblueprint.org/"
+          image: "chapters/waterloo.png"
+        - name: UCLA
+          description: Blueprint at the University of California, Los Angeles was established in 2019 🚀
+          url: "https://lablueprint.org/"
+          image: "chapters/ucla.png"
+        - name: University of Toronto 
+          description: We were very excited to establish the second Canadian chapter in 2020 🌿   
+          url: "http://uoftblueprint.org/"
+          image: "chapters/toronto.png"
+        - name: UC Irvine
+          description: Our second Southern California Blueprint chapter was established in 2020 ♥️
+          url: "https://uciblueprint.org/#/"
+          image: "chapters/uci.png"
+        - name: Carleton University 
+          description: Our third Blueprint chapter in Canada 🍂
+          url: "https://cublueprint.org/"
+          image: "chapters/carleton.png"
+        - name: University of Alabama
+          description: We were very proud to bring yet another BP chapter in a non-tech spot ✨
```

Comment:
This "non-tech spot" wording seems a little off to me, since it doesn't feel right to judge whether they're in a "tech spot" or not. Could we possibly say something like "We are super excited to see a BP chapter all the way across the country!" or something similar to other chapters', like "Our first South-East United States chapter was established in _____!"